cmd/antrea-agent/config.go

```diff
@@ -98,6 +98,11 @@ type AgentConfig struct {
 	// APIPort is the port for the antrea-agent APIServer to serve on.
 	// Defaults to 10350.
 	APIPort int `yaml:"apiPort,omitempty"`
+
+	// ClusterMembershipPort is the server port used by the antrea-agent to run a gossip-based cluster membership protocol.
```

Comment:
Probably "server port" -> "TCP port"

Reply:
And we should mention it is used when Egress is enabled.

Reply:
done

Reply:
You missed the comment: "Currently it's used only when the Egress feature is enabled."

Reply:
done

Reply:
nit: don't mix the usage and the default value in same line.
> ClusterMembershipPort is the server port used by the antrea-agent to run a gossip-based cluster membership protocol. Currently it's used only when the Egress feature is enabled.
> Defaults to 10351.